### ISSUE TITLE
Spread library updates across the update interval

### ISSUE THREAD

**Issue by L00maca** [tachiyomiorg/tachiyomi#6968]
### Describe your suggested feature

Would it be possible as an option to make it so the updates just happen spaced along all the update interval instead of all at once?

Let's say I have 400 series in ongoing that need to be checked for updates once every 24 hours.
 If they get updated all at once often sources get cranky and the app has limited functionality for a long time until the update is done (can't manually update a single series, redownlaod a chapter or migrate until is done).
What I think would be useful is that each series is updated once every set interval, but spread across the whole interval (using the number above, one every 3-4 mintues, or maybe 16-17 every hour). 
Updates won't take that much at once and sources get flooded less in a short period. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.13.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by mpldr** [comment]
As suggested, here my "Illustrations" from my ticket:

I'd call it "rolling" updates. Meaning: only query for updates for a few manga at once every x units of time.

As an example:
Assuming we have 12 Manga at one source:
```
15:00: 1, 2, 3 updated
15:05: 4, 5, 6 updated
15:10: (internet connection cuts out/update interrupted) 8 updated
15:15: 7, 9, 10 updated
15:20: 11, 12 updated
```

This would flatten the curve for the source and would theoretically even allow for more frequent update (twice a day is still plenty though). If multiple sources are queried, these can be done in parallel (with a limit on how many at the same time to limit strain on the phone CPU/Network), like:
```
15:00: S1-1, S1-2, S1-3, S2-1, S2-2 updated
15:05: S1-4, S1-5, S1-6, S2-3, S2-4 updated
15:10: S2-5 updated
```

Sensible values for max. concurrent updates per source should probably be defined in the extension, max. concurrent updates in the app settings. (Honestly, no idea what a sensible default would be. Not that deep into Android development)
